toolkit/partner-chains-cli/src/select_utxo.rs

```diff
@@ -234,12 +144,14 @@ pub(crate) mod tests {
 
 	pub(crate) fn mock_5_valid_utxos_rows() -> Vec<String> {
 		vec![
-			"f5f58c0d5ab357a3562ca043a4dd67567a8399da77968cef59fb271d72db57bd#0 (1700000 lovelace)".to_string(),
-			"b031cda9c257fed6eed781596ab5ca9495ae88a860e807763b2cd67c72c4cc1e#0 (1500000 lovelace)".to_string(),
-			"917e3dba3ed5faee7855d99b4a797859ac7b1941b381aef36080d767127bdaba#0 (1400000 lovelace)".to_string(),
-			"76ddb0a474eb893e6e17de4cc692bce12e57271351cccb4c0e7e2ad864347b64#0 (1200000 lovelace)".to_string(),
 			"4704a903b01514645067d851382efd4a6ed5d2ff07cf30a538acc78fed7c4c02#93 (1100000 lovelace)".to_string(),
-		]
+			"76ddb0a474eb893e6e17de4cc692bce12e57271351cccb4c0e7e2ad864347b64#0 (1200000 lovelace)".to_string(),
```

Comment:
Either rename `mock_result_5_valid` and `mock_5_valid_utxos_rows` to be 8 instead of 5 or remove 3 entries from them, please.